matrix/mat64/triangular.go

```diff
@@ -418,8 +418,8 @@ func copySymIntoTriangle(t *TriDense, s Symmetric) {
 				for j := i; j < n; j++ {
 					t.mat.Data[i*ts+j] = sd.Data[j*ss+i]
 				}
-				return
 			}
+			return
```

Comment:
Done. The test fails on master, passes with this change. But since RawSymmetricer has data stored in the upper triangle, the bad code path would never be taken.